### ISSUE TITLE
Undefined variable: "$golden"

### ISSUE THREAD

**Issue by stoplion** [modularscale/modularscale-sass#35]
Hello, having some problems with installing as a gem.

I've bundled the Gem, added modular-scale to compass.rb (Rails project)

In applications.css.sass I've added `@import "modular-scale"` and then in another file called..

```
$ratio: $golden
```

The @import is definitely being called before $ratio is set.  
Getting a `Undefined variable: "$golden"` 

_Edit_
Getting the same error using it inline, while removing the global $ratio var.
`font-size: ms(3, 16, $golden)`

**Comment by rpkoller** [comment]
Golden shouldn't be declared as a variable in that context. I guess you should use: 

$ratio: golden();

**Comment by stoplion** [comment]
@rpkoller Thanks for the reply. 

While inspecting the html output in Chrome I noticed the `font-size: modular-scale(3, 16px)` is literally being spit out as `font-size: modular-scale(3, 16px)`. Very strange. I think like I've followed the directions pretty thoroughly, and done the usual with installing a Compass plugin in Rails..
1. Gem install modular-scale
2. Add gem modular-scale to Gemfile and bundle it.
3. In application.sass (manifest file in Rails) @import modular-scale
4. require modular-scale in config/compass.rb
5. Restart server
